apps/model/model.js

```diff
@@ -154,6 +154,13 @@ async function initUIcomponents() {
     headerText: 'Help',
     hasFooter: false,
   });
+  // Create Modal to take input from user of new class list
+  $UI.chngClassLst = new ModalBox({
```

Comment:
chngClassLst UI component has already been defined just above that. It's a mistake I think.

Reply:
Thanks for pointing it out. It was by mistake.